src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final Scanner in = new Scanner(System.in);
+
+    /**
+     *
+     * List of all Tasks
+     */
+    private static Task[] allTasks;
+
+    private static String input;
+    private static String logo = getLogo();
+    private static String line = getLine();
+    private static int inputCount = 0;
+
     public static void main(String[] args) {
+        initTaskList();
+        welcomeMessage(logo, line);
+        while(true) {
+            String userInput = getUserInput();
+            taskQuery(userInput);
+        }
+    }
+
+    private static String getUserInput() {
+        input = in.nextLine();
+        System.out.println(line);
+        return input;
+    }
+
+    private static void taskQuery(String userInput) {
+        if(userInput.equals("bye")) {
+            System.out.println("Bye, see you!\n" + line);
+            System.exit(0);
+        }
+        else if(userInput.equals("list")) {
+            for(int i = 0; i < inputCount; i++){
+                //System.out.println((i+1) + ". [" + inputs[i].getStatusIcon() + "] " +
+                System.out.println((i+1) + "." + allTasks[i]);
+            }
+        }
+        //assumes user will follow format "done x"
+        else if(userInput.startsWith("done ")) {
```

Comment:
is there a case where user tries to mark a task not in the list as done? e.g. list has 2 items and user does 'done 5'